src/protocol.h

```diff
@@ -446,8 +448,8 @@ class CInv
     bool IsMsgBlkOrMsgWitnessBlk() const { return type == MSG_BLOCK || type == MSG_WITNESS_BLOCK; }
     bool IsGenBlkMsg() const { return type == MSG_BLOCK || type == MSG_FILTERED_BLOCK || type == MSG_CMPCT_BLOCK || type == MSG_WITNESS_BLOCK; }
 
-    uint32_t type;
     uint256 hash;
+    void SetTypeBitwiseOr(uint32_t flags) { type |= flags; }
```

Comment:
Does it make sense to do some sanity checking of the supplied `flags` here?

Reply:
I'm looking at removing this completely (per https://github.com/bitcoin/bitcoin/pull/19596#discussion_r462801556).

Reply:
Removed unused `nFetchFlags` from TX and INV net processing in ProcessMessages().